src/accessibility.md.njk

```diff
@@ -38,12 +38,9 @@ The Design System and Frontend were introduced in June 2018 to replace the follo
 
 The GOV.UK Design System team no longer supports these products and will not be making updates to help them meet level AA of WCAG 2.1. 
 
-If you are using these products, it’s recommended that you begin [updating your service now to use the Design System and Frontend](https://design-system.service.gov.uk/get-started/updating-your-code/).
+If you are using these products, you should start [updating your service to use the Design System and Frontend](https://design-system.service.gov.uk/get-started/updating-your-code/). Read guidance on [how to get started](https://design-system.service.gov.uk/get-started/).
```

Comment:
The second link is the parent of the first link, which might lead to a confusing user journey. Could we do without the second link?

Reply:
Yes we could. I'll take it out.